ci-operator/step-registry/cucushift/installer/check/azure/private/cucushift-installer-check-azure-private-commands.sh

```diff
@@ -89,6 +89,27 @@ if [[ -z ${public_ip} ]]; then
     critical_check_result=1
 fi
 
+if test -f "${SHARED_DIR}/proxy-conf.sh"
+then
+    # shellcheck disable=SC1090
+    source "${SHARED_DIR}/proxy-conf.sh"
+fi
+
+#nsg rule 'apiserver_in' check
+ocp_minor_version=$(oc version -o json | jq -r '.openshiftVersion' | cut -d '.' -f2)
+if (( ${ocp_minor_version} > 12 )); then
+    echo "Checking that nsg rule 'apiserver_in' is not created when cluster is installed in existing vnet"
```

Comment:
In theory, the checking can be applied to all the customer vpc cases, right?

Reply:
correct, so I choose one kind of jobs to cover it.

Reply:
I guess we can separate the checking into a step, so that we can opt in the checking into any customer vnet cases per our demands.

Reply:
moved out to separate step